### PR TITLE
ClusterLoader - Updating pod startup measurement logs

### PR DESCRIPTION
**Pull request by krzysied**
Added:
- failing when podStartupLatencyThreshold(5s) is not satisfied.
- logging pod latencies.

ref #293

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go [hunk 1]
```diff
@@ -207,6 +221,18 @@ func (p *podStartupLatencyMeasurement) gather(c clientset.Interface) ([]measurem
 	var err error
 	if successRatio := float32(len(startupLag)) / float32(len(p.createTimes)); successRatio < successfulStartupRatioTreshold {
 		err = fmt.Errorf("only %v%% of all pods were scheduled successfully", successRatio*100)
+		glog.Errorf(err.Error())
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go]: I think glog.Errorf(err) is what you want

File: clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go [hunk 2]
```diff
@@ -207,6 +221,18 @@ func (p *podStartupLatencyMeasurement) gather(c clientset.Interface) ([]measurem
...
+
+	podStartupLatencyThreshold := &measurementutil.LatencyMetric{
+		Perc50: podStartupLatencyThreshold,
+		Perc90: podStartupLatencyThreshold,
+		Perc99: podStartupLatencyThreshold,
+	}
+
+	if slosErr := podStartupLatency.E2ELatency.VerifyThreshod(podStartupLatencyThreshold); slosErr != nil {
+		err = measurement.NewMetricViolationError("pod startup", "there should be no high-latency requests")
+		glog.Errorf(err.Error())
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go]: same here

File: clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go (hunk 1 above)
Comment (krzysied) [reply clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go]: Fixed.

File: clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go (hunk 2 above)
Comment (krzysied) [reply clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go]: Done.

**Pull request merged**